### DISCUSSION TITLE
Add Velocity support for CMIB

### DISCUSSION THREAD

**Discussion by AZFire1** [Zrips/CMI#5758] [Feature Requests]
Velocity - https://velocitypowered.com/wiki/
Velocity Developer API - https://velocitypowered.com/wiki/developers/creating-your-first-plugin/

**Comment by CloudeLecaw** [comment]
https://github.com/Zrips/CMI/discussions/2515

**Comment by mrfloris** [comment]
https://www.spigotmc.org/resources/cmi-298-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/update?update=521168

There is a mention in the changelog for velocity support in today's CMI release.
You can get the free cmiv jar file here: https://www.zrips.net/cmiv/
